apiserver/facades/client/sshclient/shim.go

```diff
@@ -36,10 +38,19 @@ type NetworkService interface {
 	GetAllSubnets(ctx context.Context) (network.SubnetInfos, error)
 }
 
+// ApplicationService is the interface that is used to interact with the
+// applications.
+type ApplicationService interface {
+	// GetUnitMachineName gets the name of the unit's machine.
+	//
+	// The following errors may be returned:
+	//   - [applicationerrors.UnitNotFound] if the unit cannot be found.
+	GetUnitMachineName(context.Context, unit.Name) (machine.Name, error)
+}
+
```

Comment:
Since you're here, move these services (Application + Network) to facade.go, this shim must disappear.